_spinnaker/terraform_integration.md

```diff
@@ -316,7 +408,7 @@ ${#stage('My Output Stage')["context"]["status"]["outputs"]["bucket_arn"]["value
 
 ## Configuring Terraform for your cloud provider
 
-Since the Terraform Integration executes all Terraform commands against the `terraform` binary, all methods of configuring authentication are supported for your desired cloud provider. This section describes how to accomplish this for various cloud providers.
+Since the Terraform Integration executes all Terraform commands against the `terraform` binary, all methods of configuring authentication are supported for your desired cloud provider. This section describes how to configure this for AWS specifically.
 
 ### Configuration for AWS
```

Comment:
Can the shared credentials file stuff be removed w/ the addition of named profiles?

Reply:
I believe it can, yes!